include/mbedtls/config.h

```diff
@@ -238,6 +238,7 @@
 //#define MBEDTLS_BLOWFISH_ALT
 //#define MBEDTLS_CAMELLIA_ALT
 //#define MBEDTLS_DES_ALT
+//#define MBEDTLS_ECDH_ALT
```

Comment:
You are providing option to replace two functions, therefore I think it would be more appropriate to use a `MBEDTLS__FUNCTION_NAME__ALT` type macro for each instead of a single `MBEDTLS__MODULE_NAME__ALT` type flag.

Reply:
I understand your point, and I will work on it, although I don't see any other way to have an alternative implementation for ECDH, since all other functions are related to tls , and other operations not related specifically to the cryptographic operation